src/blocks/frontend/live-search/index.ts

```diff
@@ -0,0 +1,363 @@
+/**
+ * Wordpress dependencies
+ */
+// @ts-ignore
+import { debounce } from 'lodash';
```

Comment:
It's better to isolate this one function or load it using npm as this one will enqueues entire lodash.js file to the frontend.

Reply:
https://www.npmjs.com/package/lodash.debounce